zokrates_core/src/proof_system/bellman/mod.rs

```diff
@@ -162,8 +162,19 @@ impl<T: BellmanFieldExtensions + Field> Prog<T> {
 }
 
 impl<T: BellmanFieldExtensions + Field> Computation<T> {
+    fn get_random_seed(&self) -> Result<[u32; 8], getrandom::Error> {
+        let mut seed = [0u8; 32];
+        getrandom::getrandom(&mut seed)?;
+
+        use std::mem::transmute;
+        let seed: [u32; 8] = unsafe { transmute(seed) };
```

Comment:
Is there a way to achieve this without an unsafe block? If not, a comment explaining why this is safe would be good here.

Reply:
I could not find any other neat native solution for this but I believe this code is safe in this case imo, and it's the fastest. It just reinterprets a slice of u8's to a slice of 8 u32 unsigned integers. Byte order or anything does not matter here, as it's used as a seed for the rng. I feel like this code is quite clear for what it does, but I can leave a comment in the code if you feel like this is needed.

Reply:
Yes if you can put what you just explained here in there that would be nice! "This is safe because ..."